entries/data.xml

```diff
@@ -31,6 +31,7 @@ $( "body" ).data(); // { foo: 52, bar: { myType: "test", count: 40 }, baz: [ 1,
       </code></pre>
       <p>In jQuery 1.4.3 setting an element's data object with <code>.data(obj)</code> extends the data previously stored with that element.</p>
       <p>Prior to jQuery 1.4.3 (starting in jQuery 1.4) the <code>.data()</code> method completely replaced all data, instead of just extending the data object. If you are using third-party plugins it may not be advisable to completely replace the element's data object, since plugins may have also set data.</p>
+      <p>jQuery 3 changes the behavior of this method to align it to the <a href="http://www.w3.org/TR/html5/dom.html#dom-dataset">Dataset API specifications</a>. Specifically, jQuery 3 transforms all the properties' key to be camelCase. Writing a statement like <code>$( "body" ).data( {"my-name": "aValue"} ).data();</code> will return <code>{ myName: "aValue" }</code>.</p>
```

Comment:
Every two-character sequence of "-" (U+002D) followed by a lowercase ASCII letter is replaced by the uppercase version of the letter. It's the algorithm defined at [http://www.w3.org/TR/html5/dom.html#dom-dataset](http://www.w3.org/TR/html5/dom.html#dom-dataset).